static/index.html

```diff
@@ -420,22 +420,41 @@ <h2>Measurements</h2>
         placeholder: "Grammar",
       });
 
+      function encodeShareData(data) {
+        return btoa(JSON.stringify(data));
+      }
+
+      function decodeShareData(encoded) {
+        return JSON.parse(atob(encoded));
+      }
+
+      function shareURL() {
+        const gdata = encodeShareData(shareData());
+        const url = new URL(window.location);
+        url.searchParams.set("gdata", gdata);
+        url.hash = "editor";
+        return url.toString();
+      }
+
+      function copyShareLink() {
+        navigator.clipboard.writeText(shareURL())
+        const copyButton = document.getElementById("shareLinkBtn");
+        copyButton.classList.add("copied");
+        const originalText = copyButton.innerText;
```

Comment:
Just a nit - if someone clicks twice, the `originalText` will be `Copied` and the actual text will not be restored. Not a big deal though :)

Reply:
Oh good catch! I'll fix that.